docs/submission.md

```diff
@@ -1,4 +1,89 @@
+
+## To Submit - Submission Form
+1. Log into https://osparc.io/
+2. Navigate to the *Discover* tab
+3. In the *Services* section (under *Templates*) click ``Submit New Service``
+4. Fill in required information according to the descriptions outlined below
+5. Click ``Submit``
+6. Wait for your favorite team to get back to you
+
+![submission](../_media/submit.gif)
+## What You Should Include in Your Submission
+This section explains in detail what to include with the submission form introduced above.
+### Code
+If you have an interpreted language (MATLAB/Python) you can directly give us the code files. Otherwise, give us the binaries or the executable. If you are already using Docker, you may provide us your Docker images
+**Code Transfer Options**
+* Git-based public repo is the preferred option so that we can keep track of the exact version. 
+* Alternatives to Github (e.g. Bitbucket, Sourceforge, Dockerhub etc.) are also possible
+* Attachment via .zip file to our submission form (10Mb limit)
+
+### Code Metadata
+Preferably in README submitted with your code repository/zip file and submission form.
+
+**Code Requirements**
+
+For interpreted languages: 
+* Version of the interpreter (e.g. Python 3, MATLAB R2017a) 
+* Dependencies on modules/libraries (Python) or special toolboxes (MATLAB)
+
+For binaries/executables: 
+* Native OS 
+* Dependencies on external libraries 
+
+For all: 
+* Command line interface (CLI), the text-based command that runs your code using specified inputs
+
+### Validation Data
+To be submitted with your code repository/zip file and submission form.
+* Input parameters taken by your CLI and the outputs that the code produces
+* Example: for input time=200, the CLI python3 run_timer.py ${time} should produce a text file list.txt with integer values from 0 to 200
+
+
+### User Metadata via Submission Form
+**Contact Information:** to give credit where credit is due
+
+**Description:** let users know what your code does, (*e.g. This submission models transmembrane voltage of the XX nucleus in response to single-pulse stimulation of YY based on stimulation parameters ZZ*)
+
+**Inputs/outputs:**
+* name: variable name consistent with your CLI
+* type: type of variable (e.g. csv file, integer, string etc.)
+* label: name that the user sees
+* default value: to give the user a clue 
+* units: units of measure (*e.g. millivolts/mV, seconds/s, etc.*)
+* description: what the variable is or what it means
+
+
+## Code Organization for Interpreted Languages (Python & MATLAB)
+We also require a very loose code structure for interpreted language code submissions. These requirements are outlined below and if you need any help in implementation, do not hesitate to contact us at support@osparc.io 
+
+We require a “main” function that:
```

Comment:
MINOR: see how ``“main”`` vs ``` ``main`` ``` renders 

- “main”
- ``main`` : i like thisone for computer code :-)

Reply:
changed - thanks for the review!